sdks/java/core/src/test/java/org/apache/beam/sdk/io/TFRecordIOTest.java

```diff
@@ -287,6 +288,18 @@ public void runTestRoundTripZlibFilesWithAuto() throws IOException {
     runTestRoundTrip(LARGE, 10, ".tfrecords", DEFLATE, AUTO);
   }
 
+  @Test
+  @Category(NeedsRunner.class)
+  public void runTestRoundTripLargeRecords() throws IOException {
```

Comment:
Do both of the new tests fail or just the GZIP version without this fix? (just curious).

Reply:
runTestRoundTripLargeRecordsGzip() fails without this change. Other test is for completeness.